### PR TITLE
Stop bubbling of drop events for drag and drop

### LINKED ISSUE #1952: Event bubbling with dnd
# 🐛 Bug Report 

When there are nested droppable elements with spectrum dnd, only the child element event should be propagated and the parent elements' events should be stopped by default.

## 🤔 Expected Behavior

`onDrop` should get executed only for the element where it is dropped.

## 😯 Current Behavior

`onDrop` is being executed for all the parent/droppable elements.

## 💁 Possible Solution

Stopping the event propagation by default

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3.10.0
| Browser          | Chrome 90.0.4430.212 
| Operating System | MacOS

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes #1952 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Play around with the nested drop regions story. When dragging into/out of/ hovering while dragging/dropping into the child droppable region, you shouldn't see multiple onDrops/onDropActivate/onDropEnter/onDropExits happen

## 🧢 Your Project:

RSP

### CODE REVIEW

File: packages/@react-aria/dnd/stories/dnd.stories.tsx [hunk 1]
```diff
@@ -65,6 +65,17 @@ storiesOf('Drag and Drop', module)
       </Flex>
     )
   )
+  .add(
+    'nested drop regions',
+    () => (
+      <Flex direction="column" gap="size-200" alignItems="center">
+        <Draggable />
+        <Droppable>
```
[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-aria/dnd/stories/dnd.stories.tsx]: Good idea, I've added a prop to Droppable which appends what the user passes in to the action name.

**Review by snowystinger — COMMENTED** [review]
Looks good, one oddity I noticed is that I can't keyboard navigate between the child and parent. Is that part of this PR?

**Review by snowystinger — APPROVED** [review]
Needed a page refresh, @LFDanLu is opening an issue about it going to the child first. LGTM in chrome/safari/ff

**Comment by reidbarber** [comment]
LGTM

**Review by devongovett — APPROVED** [review]

**Pull request merged**